odtp/cli/execution.py

```diff
@@ -84,26 +87,11 @@ def run(
         flowManager = WorkflowManager(execution, project_path, secrets)
         flowManager.run_workflow()
     except Exception as e:
-        print(f"ERROR: Run execution failed: {e}")       
+        log.error(f"ERROR: Prepare execution failed: {e}")       
         raise typer.Abort()
     else:
-        print("SUCCESS: containers for the execution have been run")      
-
+        log.info("SUCCESS: containers for the execution have been run")      
 
-@app.command()
-def output(
```

Comment:
Please don't remove this method, since I am using it from the `gui`

Reply:
OK, Sorry. Rolling back.  Maybe we can find a better name for this method. Originally, this method was conceived to retrieve the outputs from any step in the execution. i.e. Download the s3 compressed output.